fflib/src/classes/fflib_QueryFactory.cls

```diff
@@ -543,6 +550,79 @@ public class fflib_QueryFactory { //No explicit sharing declaration - inherit fr
 		);	
 		return this;
     }
+	/**
+	 * Remove existing ordering and set a field to be sorted on.  This may be a direct field or a field 
+	 * related through an object lookup or master-detail relationship.
+	 * Use the set to store unique field names, since we only want to sort
+	 * by the same field one time.  The sort expressions are stored in a list
+	 * so that they are applied to the SOQL in the same order that they
+	 * were added in. 
+	 * @param fieldName The string value of the field to be sorted on
+	 * @param SortOrder the direction to be sorted on (ASCENDING or DESCENDING)
+	 * @param nullsLast whether to sort null values last (NULLS LAST keyword included).
+	**/  
+    public fflib_QueryFactory setOrdering(String fieldName, SortOrder direction, Boolean nullsLast){
+		setOrdering(
```

Comment:
I would just return setOrdering, rather than having return on a separate line.
```java
Ordering o = new Ordering(getFieldTokenPath(fieldName), direction, nullsLast);
return setOrdering(o);
```

Also, I think the method should be **getFieldTokenPath**, not **getFieldToken**

Reply:
Deploy is now failing with these errors:
```
classes/fflib_QueryFactory.cls(586,17):Method does not exist or incorrect signature: void getFieldToken(String) from the type fflib_QueryFactory
classes/fflib_QueryFactory.cls(623,17):Method does not exist or incorrect signature: void getFieldToken(String) from the type fflib_QueryFactory
classes/fflib_QueryFactoryTest.cls(175,10):Comparison arguments must be compatible types: Schema.SObjectField, String
classes/fflib_QueryFactoryTest.cls(182,10):Comparison arguments must be compatible types: Schema.SObjectField, String
classes/fflib_QueryFactoryTest.cls(189,10):Comparison arguments must be compatible types: Schema.SObjectField, String
classes/fflib_QueryFactoryTest.cls(196,10):Comparison arguments must be compatible types: Schema.SObjectField, String
classes/fflib_QueryFactoryTest.cls(203,10):Comparison arguments must be compatible types: Schema.SObjectField, String
classes/fflib_SObjectSelector.cls(586,17):Dependent class is invalid and needs recompilation:
 Class fflib_QueryFactory : Method does not exist or incorrect signature: void getFieldToken(String) from the type fflib_QueryFactory
classes/fflib_SObjectSelectorTest.cls(586,17):Dependent class is invalid and needs recompilation:
 Class fflib_SObjectSelector : Dependent class is invalid and needs recompilation:
 Class fflib_QueryFactory : Method does not exist or incorrect signature: void getFieldToken(String) from the type fflib_QueryFactory
classes/fflib_ApplicationTest.cls(586,17):Dependent class is invalid and needs recompilation:
 Class fflib_SObjectSelector : Dependent class is invalid and needs recompilation:
 Class fflib_QueryFactory : Method does not exist or incorrect signature: void getFieldToken(String) from the type fflib_QueryFactory
```